schemas/content/product.schema.json

```diff
@@ -17,9 +18,8 @@
         "@id": {
           "title": "Identifier",
           "type": "string",
-          "format": "uri",
           "description":
-            "The internal unique ID of the variant in the commerce backend system."
+            "The internal unique ID of the product or variant in the commerce backend system."
```

Comment:
"commerce backend system" is not always right. t could be from a PIM etc. Best to not be that explicit.

Reply:
Already updated the description.